spec/features/training_tool_spec.rb

```diff
@@ -101,6 +101,17 @@
       expect(tmu.reload.last_slide_completed).to eq(module_2.slides.second.slug)
     end
 
+    it 'allows for navigation with the left and right buttons' do
+      click_link 'Start'
+      sleep 1.5
+      tmu = TrainingModulesUsers.find_by(user_id: user.id, training_module_id: module_2.id)
+      expect(find('body').text).to have_content('Page 1 of')
```

Comment:
Is there any benefit to `find('body').text` over `page` here? I usually use `expect(page).to have_content`, just because it is closer to English.